notebooks/BIA_SCRAPER.ipynb

```diff
@@ -642,446 +642,460 @@
         "id": "4AXvG3fFeJaz"
```

Comment:
Is anything changed in the class? It looks like it's all the same but for some reason github is marking it all as deleted